hera_mc/cm_dataview.py

```diff
@@ -27,149 +25,43 @@ def __init__(self, session=None):
             self.session = db.sessionmaker()
         else:
             self.session = session
-        self.parts_list = None
-        self.fc_map = None
 
-    def print_fully_connected(self, fully_connected):
+    def ants_by_day(self, start, stop, time_step, output=None, station_types_to_check='default', output_date_format='jd'):
         """
-        Prints out correlator information from 'standard' fully_connected list
+        Return dated list of connected stations.
 
         Parameters:
         ------------
-        fully_connected:  list of fully connected stations provided by the correlator
-        methods in geo_handling
+        start:  start (astropy.Time)
+        stop:  stop (astropy.Time)
+        time_step:  time_step in days between start/stop (float or int)
```

Comment:
Sorry, this is still confusing to me. It looks like you want me to give you the number of days between the start & stop values (which you can just compute). Actually, I think it's the cadence at which you check which ants are connected. I do like the name `time_step` better than `interval`.

Reply:
I think the problem for me is the phrase `between start/stop'. If you clarify the docstrings on start and stop you can probably take that phrase out.